website/docs/r/organization_default_settings.html.markdown

```diff
@@ -20,14 +20,20 @@ resource "tfe_organization" "test" {
 }
 
 resource "tfe_agent_pool" "my_agents" {
-  name = "agent_smiths"
+  name         = "agent_smiths"
   organization = tfe_organization.test.name
 }
 
-resource "tfe_organization_default_execution_mode" "org_default" {
-  organization = tfe_organization.test.name
+resource "tfe_organization_default_settings" "org_default" {
+  organization           = tfe_organization.test.name
   default_execution_mode = "agent"
-  default_agent_pool_id = tfe_agent_pool.my_agents.id
+  default_agent_pool_id  = tfe_agent_pool.my_agents.id
+}
+
+resource "tfe_workspace" "my_workspace" {
+  name       = "my-workspace"
+  # Ensures this workspace will inherit the org defaults
+  depends_on = [tfe_organization_default_settings.org_default]
```

Comment:
(Recording this for posterity, no one needs to respond to this.) I was confused about why this was necessary, because the workspace will just inherit the default anyway if you don't specify an overridden value. But the issue is that the workspace reports the inherited default as the value of its attribute of the same name, so without the ordering, you get needless churn — the workspace first reports it inherited the old default, then on the second run it reports that it inherited the new default as a change to the computed value. So the ordering _is_ important, because it'll give you a stable result on the first apply. 👍🏼